src/EnergyPlus/IceThermalStorage.cc

```diff
@@ -2043,7 +2043,7 @@ namespace IceThermalStorage {
             } else { // (DetailedIceStorage(IceNum)%InletTemp < DetailedIceStorage(IceNum)%OutletTemp) Discharging Mode
 
                 this->DischargingRate = this->CompLoad;
-                this->DischargingEnergy = this->CompLoad * (state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour);
+                this->DischargingEnergy = this->CompLoad * (state.dataHVACGlobal->TimeStepSys * Constant::SecInHour);
```

Comment:
Wow, there are lots of these.

Reply:
Yes. Apparently `sed "s/TimeStepSys\s\*\sDataGlobalConstants::SecInHour/TimeStepSysSec/g"` did not work. The `\*` messed it up. I will have to look up some regex magic on StackOverflow.